validation/translations.rst

```diff
@@ -135,5 +135,65 @@ Now, create a ``validators`` catalog file in the ``translations/`` directory:
             'author.name.not_blank' => 'Please enter an author name.',
         ];
 
-You may need to clear your cache (even in the dev environment) after creating this
-file for the first time.
+You may need to clear your cache (even in the dev environment) after creating
+this file for the first time.
+
+Custom Translation Domain
+-------------------------
+
+The default translation domain can be changed globally using the
+``FrameworkBundle`` configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/translation.yaml
+        framework:
+            translator:
+                translation_domain: validation_errors
```

Comment:
I haven't checked but ... is this the full config needed? Because in the config option names I don't see anything about validation. This looks like you are changing the translation domain for everything, instead of only for the validation feature. Thanks.

Reply:
Oops, you're right indeed.

Reply:
Fixed in https://github.com/symfony/symfony-docs/pull/18873/commits/e6ace65f50870d4f1e7169c5681d53f91593e9ba.